lib/constable_web/templates/slack_channel/edit.html.eex

```diff
@@ -22,25 +22,24 @@
             placeholder: gettext("e.g. #everyone")
           ) %>
         </div>
-        <div class="tbds-block-stack__item">
+        <div class="tbds-block-stack__item flex-button-container">
           <%= submit(
             gettext("Save Slack Channel"),
             id: "submit-channel-name",
             class: "tbds-button"
           ) %>
+          <%= if @interest.slack_channel do %>
+            <%= link(
+              to: Routes.interest_slack_channel_path(@conn, :delete, @interest),
+              method: :delete,
+              data: [confirm: gettext("Are you sure?"), role: "remove-channel"],
+              class: "tbds-button button-secondary remove-slack-channel"
```

Comment:
It looks like we removed the `remove-slack-channel` CSS above - can this class be removed here?

```suggestion
              class: "tbds-button button-secondary"
```